packages/tests/server/react/__reactHelpers.tsx

```diff
@@ -36,7 +36,11 @@ export function getServerAndReactClient<TRouter extends AnyRouter>(
   });
 
   const queryClient = createQueryClient();
-  const proxy = createTRPCReact<typeof appRouter>();
+  const proxy = createTRPCReact<
+    typeof appRouter,
+    unknown,
+    'ExperimentalSuspense'
+  >();
```

Comment:
This is how you enable it

Reply:
should feature flags be an array?

Reply:
I'd also want autocomplete on them. Types became annoying. Feel free to have a stab at improving it

Reply:
It doesn't need to be an array, it could work like the second arg of `Pick` etc

Reply:
Ahh I read the `'ExperimentalSuspense'` argument as runtime code the first time but it's passed as a generic